integration/util.py

```diff
@@ -924,6 +992,11 @@ def twisted_sleep(reactor, timeout):
     return deferLater(reactor, timeout, lambda: None)
 
 
+def sleep(timeout):
```

Comment:
This most definitely begs for a docstring and maybe even a more descriptive name.